dataactcore/utils/report.py

```diff
@@ -1,18 +1,22 @@
 import itertools
 from operator import attrgetter
 
-from dataactcore.models.lookups import FILE_TYPE
+from dataactcore.models.lookups import FILE_TYPE, FILE_TYPE_DICT_NAME_LETTER
 
 
 def report_file_name(submission_id, warning, file_type, cross_type=None):
     """Format the csv file name for the requested file.
     @todo: unify these file names"""
     if cross_type:
         report_type_str = 'warning_' if warning else ''
-        return "submission_{}_cross_{}{}_{}.csv".format(submission_id, report_type_str, file_type, cross_type)
+        file_letters = 'File_{}_to_{}_'.format(FILE_TYPE_DICT_NAME_LETTER[file_type],
```

Comment:
Since there is no longer an `if` is this variable really necessary?

Reply:
I kept it in for readability but can put it directly in the return